### ISSUE TITLE
22 is allowed, 23 is not. but 22 is cheaper than 23.

### ISSUE THREAD

**Issue by elden1337** [elden1337/hass-peaq#218]
avg month: 1.15 SEK
prices today: [0.463, 0.491, 0.51, 0.538, 0.617, 1.262, 1.521, 1.913, 1.877, 1.579, 1.416, 1.284, 1.198, 1.101, 1.129, 1.191, 1.309, 1.397, 1.417, 1.397, 1.347, 1.242, 0.669, 0.601]
prices tomorrow: [0.475, 0.457, 0.442, 0.42, 0.414, 0.415, 0.44, 0.466, 0.47, 0.459, 0.46, 0.456, 0.455, 0.452, 0.453, 0.422, 0.417, 0.411, 0.405, 0.372, 0.339, 0.327, 0.313, 0.257]
nonhours: 4, 15, 16, 17, 18, 19, 20, 21, 23, 0⁺¹, 1⁺¹, 7⁺¹, 8⁺¹, 9⁺¹, 10⁺¹
cautionhours: '13': 42% '22': 53%
time now: 13:48
type: SCROOGE

**tldr: 23 is more expensive than 22. Why is that one in non-hours and 22 in caution-hours?**
